### PR TITLE
Kh feature dummycomponent

### PR DESCRIPTION
**Pull request by khuang-dev**

### CODE REVIEW

**Review by KieranDevlin — COMMENTED** [review]
Please remove .DS_Store from root

File: client/js/components/CustomText/CustomText.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {Text, StyleSheet} from 'react-native';
+
+export default class CustomText extends React.Component {
```
Comment (siddharthparmar7) [inline client/js/components/CustomText/CustomText.js:4]: this should be a stateless component.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

File: client/js/assets/fonts/OFL.txt [hunk 2]
```diff
@@ -0,0 +1,94 @@
+Copyright (c) 2010-2015, Łukasz Dziedzic (dziedzic@typoland.com),
+with Reserved Font Name Lato.
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
```
Comment (siddharthparmar7) [inline client/js/assets/fonts/OFL.txt:4]: No need to add this file. Remove it.

File: client/js/assets/fonts/README.txt [hunk 3]
```diff
@@ -0,0 +1,72 @@
+
```
Comment (siddharthparmar7) [inline client/js/assets/fonts/README.txt:1]: same as above

**Pull request merged**